examples/headsail-bsp/src/alloc/mod.rs

```diff
@@ -1,12 +1,9 @@
 use good_memory_allocator::SpinLockedAllocator;
 
-const HEAP_START: usize = 0x30000000;
-const HEAP_SIZE: usize = 0x10000000;
-
```

Comment:
I don't mind having these here actually. The heap location can be considered a BSP feature. Either is fine.

Reply:
These changes we removed from the PR altogether